contrib/review-trailers-tooling/NVIM_RUNTIME_PATH/UltiSnips/gitcommit.snippets

```diff
@@ -0,0 +1,14 @@
+snippet wcm "Commit message for warning fix." b
+Problem    : ${0}
+Diagnostic :
+Rationale  :
+Resolution :
+endsnippet
+
+snippet "^r" "Insert Reviewed-by trailer" rb
```

Comment:
Maybe just `"Reviewed-by trailer"`? I'd think that `Insert` in a snippet description is redundant (same issue with the snippet below, if it is an actual issue).